NeedlemanWunsch.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import sys, string
+import alignment #import the file with the functions
+
+# Command-line arguments for 3 files and one integer.
+f1 = open(sys.argv[1], 'r') #first fasta/txt file
```

Comment:
sounds good, i'm ready to accept this, but one small nit:

for readability, please assign all argv[...] to variables at the beginning of the program, e.g.:
fasta_file_one = sys.argv[1]
fasta_file_two = sys.argv[2]
substitution_matrix_file =  sys.argv[3]
gap_penalty = sys.argv[4]

and then use the variables in the code.